mlir/test/Conversion/MemRefToEmitC/memref-to-emitc-failed.mlir

```diff
@@ -46,14 +46,6 @@ memref.global "nested" constant @nested_global : memref<3x7xf32>
 
 // -----
 
-func.func @unsupported_type_f16() {
```

Comment:
Also, keep the tests and replace the types with unsupported floating point types, like `f128`. But it's not that important. It's up to you.